src/styles/blocks/card.css

```diff
@@ -0,0 +1,60 @@
+.card {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 166px;
+  height: 356px;
```

Comment:
add 200px to card's width, use box-sizing: border-box so paddings and border will be included in this 200px